scripts/bnsd/genesis_app_state.json

```diff
@@ -194,7 +194,7 @@
     },
     "account": {
       "valid_name": "^[a-z0-9\\-_.]{3,64}$",
-      "valid_domain": "^[a-z0-9\\-_]+${3,16}",
+      "valid_domain": "^[a-z0-9\\-_]+${3,16}$",
```

Comment:
`valid_domain` should be `^[a-z0-9\\-_]{3,16}$` (no "+" and only a single "$" at the end of the string).